test/assert.js

```diff
@@ -1454,6 +1455,18 @@ describe('assert', function () {
     err(function () {
       assert.property(undefined, 'a', 'blah');
     }, "blah: Target cannot be null or undefined.");
+
+    err(function () {
+      assert.propertyVal(dummyObj, 'a', '2', 'blah');
+    }, "blah: expected { a: '1' } to have property 'a' of '2', but got '1'");
+
+    assert.doesNotThrow(function () {
+      assert.nestedProperty({a:1}, '{a:1}');
+    }, Error, 'the argument to property must be a string when using nested syntax');
```

Comment:
You don't think you need to add this test actually.
There is already a test above on line [1404](https://github.com/chaijs/chai/pull/1044/files#diff-9c6c3cd6ffcef4f57ac6e0c793b31b85R1404) for the `assert.nestedProperty` passing a String.